ui/main/src/app/modules/admin/components/table/admin-table.directive.ts

```diff
@@ -459,6 +463,12 @@ export abstract class AdminTableDirective implements OnDestroy {
     hasAnyPermission(permissions: PermissionEnum[]) {
         return UsersService.hasCurrentUserAnyPermission(permissions);
     }
+
+    onPageSizeChanged() {
+        // Cast to get rid of "Property 'value' does not exist on type 'HTMLElement'."
+        const value = (<HTMLInputElement>document.getElementById('opfab-page-size-select')).value;
+        this.dataHandlingService.changePaginationPageSize(Number(value));
+    }
```

Comment:
instead of getting the element via his id , you can take it via the method param , suggestion not tested : 

```
onPageSizeChange(element) {
const value = element.value; 
...
}

Reply:
Done.